src/ngraph/frontend/onnx_import/utils/common.hpp

```diff
@@ -38,8 +38,14 @@ namespace ngraph
     {
         namespace common
         {
-            const NodeVector& add_provenance_tags(const Node& onnx_node,
-                                                  const NodeVector& ng_node_vector);
+            /// \brief Adds a provenance tag to all nodes in a subgraph representing an ONNX node
+            void add_provenance_tags(const Node& onnx_node, const NodeVector& ng_node_vector);
+
+            /// \brief Adds a provenance tag to a node that represents an input of a graph
+            /// \note  This overload only works with Parameter and Constant nodes
+            void add_provenance_tags(const std::string& input_name,
```

Comment:
This should be named `add_provenance_tags_to_input` or just moved to the file where only inputs are tagged.